flaml/model.py

```diff
@@ -4,6 +4,7 @@
 '''
 
 import numpy as np
+from pandas.core.algorithms import isin

```

Comment:
seems like this has not been used later in this file

Reply:
Weird. I don't remember writing this line. Not sure how this line gets in. I'll remove it.